source/js/main.js

```diff
@@ -54,9 +52,24 @@ var Main = function() {
             });
         });
 
-        $('div.configuration-block').each(function (){
+        $('div.configuration-block').each(function () {
             $('li:first', $(this)).addClass('selected');
         });
+
+        $('[data-toggle="tooltip"]').tooltip();
+
+        $('button.copy-to-clipboard').on('click', function() {
+            var copyElementId = $(this).data('copy-element-id');
+
+            var copyText = $('#' + copyElementId + ' .code-line').text();
```

Comment:
This kind of concatenation is unsafe: would a callback selector still work?

Reply:
I am not familiar with what that is. Could you elaborate?